client/my-sites/media-library/data-source.jsx

```diff
@@ -90,39 +75,65 @@ export class MediaLibraryDataSource extends Component {
 				icon: <Gridicon icon="image-multiple" size={ 24 } />,
 			} );
 		}
+		return sources.filter( ( { value } ) => -1 === disabledSources.indexOf( value ) );
```

Comment:
Any reason for not using `includes` as you do [here](https://github.com/Automattic/wp-calypso/pull/28121/files#diff-0db2c9807d11b1264b1631f5e42f5dd9R80)?

Reply:
Not really. :)
I've simply copied over the [previous version](https://github.com/Automattic/wp-calypso/pull/28121/files/3536d7a776b4d011f952154be33e0537918c0948#diff-079adb61f03f78d23ab64a0a61a414f6L63), and destructured the parameter.

Though yes, it's cleaner with `includes`. I'll update ASAP.